TANMEDIAGROUP/SYNDEX_BN
# Docker configuration

### What does this PR do?

This pull request (PR) removes the `.env` file, `Dockerfile`, and `docker-compose` files from the existing codebase.

### Description of Task to be completed?

The main task involved in this PR is the removal of the `.env` file, which typically contains environment variables specific to the project's development or production environments. Additionally, the `Dockerfile` and `docker-compose` files, used for containerizing the application with Docker, have been removed.

#### tasks

- [x] Remove `.env` file from the project
- [x] Remove `Dockerfile`
- [x] Remove `docker-compose.yml` file
- [x] Update `.gitignore` file to exclude `.env` files
- [x] Update documentation or README file to reflect the changes

### how to test

- Step 1: Clone the repository or fetch the latest changes
- Step 2: Verify that the `.env`, `Dockerfile`, and `docker-compose.yml` files are no longer present in the project directory
- Step 3: Build and run the application without Docker to ensure it still functions correctly

### Any background context you want to provide?

The removal of these files may be due to security concerns, as the `.env` file often contains sensitive information like API keys, database credentials, or other environment-specific configurations. Additionally, the removal of the Docker-related files (`Dockerfile` and `docker-compose.yml`) could indicate a shift away from using Docker for deployment or development purposes.

### What are the relevant Github Issues if any?

N/A

#### Tracker id:

N/A
